packages/esm-bed-management-app/src/ward-with-beds/ward-with-beds.component.tsx

```diff
@@ -114,6 +118,25 @@ const WardWithBeds: React.FC = () => {
               }>
               <span>Return to summary</span>
             </Button>
+            <span className={styles.backgroundDataFetchingIndicator}>
```

Comment:
The stylesheet `ward-with-beds.scss` does not have styles for `backgroundDataFetchingIndicator`.